### PR TITLE
Feature/four 14796: Cancel in embed is not working launchPad

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
The Cancel button is not working the modal remains open

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14796

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-68ed736fdb.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-68ed736fdb.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-68ed736fdb.engk8s.processmaker.net

File: resources/js/components/shared/InputImageCarousel.vue [hunk 1]
```diff
@@ -583,4 +640,12 @@ export default {
...
\ No newline at end of file
+</script>
+
+<style lang="css" scoped>
+#invisiblehr {
+  margin: 0px;
+  width: 0.1px;
+  border-color: white;
+}
+</style>
```
Comment (fagubla) [inline resources/js/components/shared/InputImageCarousel.vue:651]: add a line blank at the end

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by pmPaulis — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-68ed736fdb.engk8s.processmaker.net

**Pull request merged**